### DISCUSSION TITLE
Creating Teams via REST API with GitHub App permissions without the need for approval

### DISCUSSION THREAD

**Discussion by kallusakar** [community/community#64204] [Apps, API and Webhooks]
### Select Topic Area

Question

### Body

Hi, I'm trying to automate GitHub teams creation in our organization based on our orgchart data. I've successfully used the API to create a team, however the team wasn't created, rather it pushed a notification to approve team creation made by the GitHub App. Is there a way to turn off this behavior and allow the GitHub App to create teams without needing to ask for permission? There are multiple teams that need to be created in a hierarchy and manual approval of each change is not possible.

**Comment by varunchandak** [comment]
@kallusakar I can see this was marked `closed as resolved`. What was the solution to this?
